package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mfcl",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
If I remember correctly, we should say this is 3.1.0 because we're adding a new component into the library. I feel like it's not a major change but it's also not a minor change either. Make sense?

Reply:
Ahh yes that makes sense. Since this is a feature it should definitely be 3.1.0 rather than 3.0.1. Thanks for clarifying this Lee!